src/FirstRun/LanguagePickerScreen/LanguagePickerScreen.tsx

```diff
@@ -1,43 +1,45 @@
 import {useNavigation} from '@react-navigation/native'
 import React from 'react'
 import {defineMessages, useIntl} from 'react-intl'
-import {StyleSheet, View, ViewProps} from 'react-native'
+import {StyleSheet} from 'react-native'
 import {SafeAreaView} from 'react-native-safe-area-context'
 
-import {Button, LanguagePicker} from '../../components'
+import {LanguagePicker} from '../../components'
 import {FirstRunRouteNavigation} from '../../navigation'
 
 export const LanguagePickerScreen = () => {
   const navigation = useNavigation<FirstRunRouteNavigation>()
-  const intl = useIntl()
+  const strings = useStrings()
 
   return (
     <SafeAreaView style={styles.safeAreaView}>
-      <LanguagePicker />
-
-      <Actions>
-        <Button
-          onPress={() => navigation.navigate('accept-terms-of-service')}
-          title={intl.formatMessage(messages.continueButton)}
-          testID="chooseLangButton"
-        />
-      </Actions>
+      <LanguagePicker
+        buttonLabel={strings.buttonLabel}
```

Comment:
these prop names break encapsulation

Reply:
With the new refactor it will be removed but whats your suggestion, please?